azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/common/audiodevicelist/AudioDeviceListView.kt

```diff
@@ -133,6 +181,8 @@ internal class AudioDeviceListView(
         return when (audioDeviceType) {
             AudioDeviceType.ANDROID -> context.getString(R.string.azure_communication_ui_setup_audio_device_android)
             AudioDeviceType.SPEAKER -> context.getString(R.string.azure_communication_ui_setup_audio_device_speaker)
+            // TODO: Resource string
+            AudioDeviceType.BLUETOOTH_SCO -> "Bluetooth"
```

Comment:
can we replace hardcoded string with Resource string

Reply:
done